misbehave.html

```diff
@@ -76,6 +76,27 @@
         behaviorDefinition = subBehaviors;
       }
 
+      behaviorDefinition.injectFrom = function(base, target) {
+        var baseElement = document.createElement(base);
+        // Grab an instance of the base template!
+        // _template is only available if the element hasn't been added to the DOM somewhere!
+        var baseTemplateInstance = Polymer.Base.instanceTemplate(Polymer.dom(baseElement).node._template);
+        
+        // And baseTemplateInstance will only have a children array if it hasn't been added to the DOM yet!
+        var self = this;
+        Polymer.dom(baseTemplateInstance).children.forEach(function(child) {
```

Comment:
What about children's children, etc.?  Hopefully there's a way we can get Polymer to do this work.  More info possibly here: https://github.com/Polymer/polymer/blob/b0733d3c07f071cb05ae88473cd07caee20ffe03/src/standard/annotations.html